guides/V1_ConfigurationAction.md

```diff
@@ -0,0 +1,78 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'configuring-app'
+is: 'experimental'
+---
+
+## Configuring an app
+
+There are two possible configuration actions for your app, `space-app-config-requested` and `team-app-config-requested`, 
+both are **Explicit Actions**. 
+You can learn more about the concepts and basics of action handlers, by reading [Actions & Events](./V1_ActionsEvents.md).
+
```

Comment:
We already describe the context returned for each of the two action types here, perhaps point them there for the details of the context/data for each of them:
https://github.com/watsonwork/watsonwork-developer-docs/pull/105/files#diff-3b524cfc6088c5d293e7c2de400513e9R60